mlflow/entities/model_registry/prompt_version.py

```diff
@@ -342,11 +342,22 @@ def format(
             kwargs: Keyword arguments to replace the variables in the template.
         """
         input_keys = set(kwargs.keys())
-        template_str = self.template if self.is_text_prompt else json.dumps(self.template)
-        for key, value in kwargs.items():
-            template_str = re.sub(r"\{\{\s*" + key + r"\s*\}\}", str(value), template_str)
-        template = template_str if self.is_text_prompt else json.loads(template_str)
 
+        def escape_backslashes(prompt: str) -> str:
```

Comment:
Could we define this outside of this function?

Reply:
Like change the interface to `escape_backslashes(prompt: str, kwargs: dict[str, Any]) -> str`?

Reply:
Sure, let me extract the function